programs/ziti-edge-tunnel/package/deb/postrm.in

```diff
@@ -26,3 +26,18 @@ fi
 if [ -L /usr/bin/@CPACK_PACKAGE_NAME@ ]; then
     unlink /usr/bin/@CPACK_PACKAGE_NAME@
 fi
+
+ucf --purge "/usr/share/polkit-1/rules.d/@ZITI_POLKIT_RULES_FILE@" || true
+ucf --purge "/var/lib/polkit-1/localauthority/10-vendor.d/@ZITI_POLKIT_PKLA_FILE@" || true
```

Comment:
in my testing, `ucf --purge` didn't remove the file. is it supposed to?

Reply:
I misread the ```ucf --purge``` documentation. Your observations are correct--the file isn't removed.